lms/djangoapps/appsembler_api/utils.py

```diff
@@ -59,7 +60,10 @@ def has_permission(self, request, view):
                present in the request and matches the setting.
                """
         api_key = getattr(settings, "EDX_APP_SEMBLER_API_KEY", None)
+        is_enable_api = configuration_helpers.get_value("ENABLE_APPSEMPLER_API", None)
```

Comment:
Please, consider to refactor: 
`EDX_APP_SEMBLER_API_KEY` to `EDX_APPSEMBLER_API_KEY`
`ENABLE_APPSEMPLER_API` to `ENABLE_APPSEMBLER_API`